WORKFLOW.md

```diff
@@ -24,74 +24,67 @@ git commit -m "first commit"
 git add another
 git commit -m "second commit"
 
-# rebase against staging to pull in any changes that have been made
+# rebase against master to pull in any changes that have been made
 # since you started your feature branch.
 git fetch
-git rebase origin/develop
+git rebase origin/master
 
 # push your local changes up to the remote
 git push
 
 # if you've already pushed changes and have rebased, your history has changed
 # so you will need to force the push
 git fetch
-git rebase origin/develop
+git rebase origin/master
 git push --force-with-lease
 ````
 
 
 ### GitHub workflow
 
-- Open a Pull Request against `staging`. Check our PR guidelines [here](https://github.com/quintoandar/butterfree/blob/master/CONTRIBUTING.md#pull-request-guideline).
-- When the Pull Request has been approved, merge using `squash and merge`, adding the Jira task number and a brief description:
-ie, `[MLOP-169] Enable stream pipelines in Butterfree`.
+- Open a Pull Request against `master`. Check our PR guidelines [here](https://github.com/quintoandar/butterfree/blob/master/CONTRIBUTING.md#pull-request-guideline).
+- When the Pull Request has been approved, merge using `squash and merge`, adding a brief description:
+ie, ` Enable stream pipelines in Butterfree`.
 - This squashes all your commits into a single clean commit. Remember to clean detailed descriptions, otherwise our git logs will be a mess.
 
-If you are unable to squash merge because of conflicts, you need to rebase against `staging` again:
+If you are unable to squash merge because of conflicts, you need to rebase against `master` again:
 
 ```
 # in your feature branch
 git fetch
-git rebase origin/staging
+git rebase origin/master
 # fix conflicts if they exist
 git push --force-with-lease
 ```
 
 
 ## Releases
 
-A release takes the changes in `staging` and applies them to `master`.
-
+The release will always occur when we change the version in the setup.py file.
 
-### Working locally
 
+### Working Locally
 
 ```
-# create a release branch from staging
-git checkout staging
+# just change the version on 
+git checkout master
 git fetch
-git pull origin staging
-git checkout -b release/0.4.0
+git pull origin master
 
 # finalize the changelog, dump the version into setup.py and update the documentation then:
 make update-docs
 git add .
-git commit -m "release 0.4.0"
+git commit -m "release <version>"
 
-# rebase against master, which we're going to merge into
+# push the new version
 git fetch
-git rebase origin/master
 git push --force-with-lease
```

Comment:
We shouldn't have the power to force push things to master. We could easily make serious mistakes this way. I still defend that we need pull requests to trigger releases.

Reply:
hotfix and normal releases intrinsically need work in the same way: merged PRs from branches `release-*` and `hotfix-*` triggers `Publish` worflow

Reply:
We decided not to have release ramifications to avoid having orphaned branches and not having versioning problems in Read the Docs.